lib/graphiti/query.rb

```diff
@@ -30,6 +30,10 @@ def links?
       end
     end
 
+    def pagination_links?
+      return !!@params[:pagination_links]
+    end
```

Comment:
Could we add a global config for this as well? I am thinking we can do this like [Links](https://github.com/graphiti-api/graphiti/blob/1683c96774b6a9ab70e9e3c4b501efd3b3c2ac3d/lib/graphiti/query.rb#L24-L31). This way you can turn on pagination links by default, OR require the client to request them.

I think this is the **very last thing** for this PR and then let's merge.